app/main.py

```diff
@@ -2,0 +1,10 @@
+from shutil import copy
+
+
+def copy_file(command: str) -> None:
+    _, source_file, destination_file = command.split(" ")
+
+    if source_file == destination_file:
+        return
```

Comment:
Code Efficiency Issue: Your function will silently do nothing if the source file is the same as the destination file. It might be more helpful to raise an error or at least print a message to let the user know why the file was not copied.